articles/multifactor-authentication/administrator/guardian-enrollment-email.md

```diff
@@ -0,0 +1,37 @@
+---
+description: Send an enrollment email (Guardian)
+---
+## Sending a Guardian Enrollment Email to a User.
+
+This allows an administrator to send an email to a particular user with a link that leads to the guardian enrollment step in order to have the user register a device with Guardian.
```

Comment:
maybe it's better if it just says "... with a link that let's the user register a device with Guardian" ?
the "guardian enrollment step in order to have the user ..." sounds complex